source/lib/cdk-textract-stack.ts

```diff
@@ -170,35 +198,93 @@ export class CdkTextractStack extends cdk.Stack {
       cloudfrontDocumentsBucketPolicyStatement
     );
 
+    const esLogGroup = new LogGroup(
+      this,
+      this.resourceName("ElasticSearchLogGroup"),
+      {
+        logGroupName: this.resourceName("ElasticSearchLogGroup")
+      }
+    );
+
+    // Elasticsearch
+    let elasticSearch;
+
     const esEncryptionKey = new kms.Key(this, "esEncryptionKey", {
       enableKeyRotation: true
     });
-    // Elasticsearch
-    const elasticSearch = new es.CfnDomain(
-      this,
-      this.resourceName("ElasticSearchCluster"),
-      {
-        elasticsearchVersion: "6.5",
-        elasticsearchClusterConfig: {
-          instanceType: "m5.large.elasticsearch"
-        },
-        ebsOptions: {
-          ebsEnabled: true,
-          volumeSize: 20,
-          volumeType: "gp2"
-        },
-        encryptionAtRestOptions: {
-          enabled: true,
-          kmsKeyId: esEncryptionKey.keyId
+
+    if (!props.isCICDDeploy) {
```

Comment:
I agree with Alex, why do we need if-else if we are having the same properties in both?

Reply:
there is a pricing difference between the two deployments that we would like to avoid for dev deploys.